src/Services/publicApiService.js

```diff
@@ -2,15 +2,30 @@ import axios from 'axios';
 
 const config = {
   headers: {
-    Group: 1, //Aqui va el ID del equipo!!
+    Group: 105, //Aqui va el ID del equipo!!
   },
 };
 
-const Get = () => {
-  axios
-    .get('https://jsonplaceholder.typicode.com/users', config)
-    .then((res) => console.log(res))
-    .catch((err) => console.log(err));
+const publicApiGet = async (route, id) => {
+  let url;
+
+  if (id === undefined || id === null) {
```

Comment:
if(!id) validates if the id is neither undefined, neither null

Reply:
(!id) also validates if the id is "", 0, false and other falsy values. In that case, if I pass and id parameter of 0, it will fulfill the condition and bring the entire list, not the specific one. Is it the desired behaviour?

Reply:
Good answer